src/index.html

```diff
@@ -11,12 +11,85 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Kalnia+Glaze&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="#home"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo company logo"
+        />
+      </a>
+
+      <nav class="main_nav-link">
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. The class 'main_nav-link' suggests styling rather than meaning; consider something like 'main-nav' instead.